test/e2e/scaffold/httpbin.go

```diff
@@ -96,3 +99,11 @@ func (s *Scaffold) newHTTPBIN() (*corev1.Service, error) {
 	}
 	return svc, nil
 }
+
+func (s *Scaffold) ScaleHTTPBIN(num int) error {
```

Comment:
Does teratest has methods like `kubectl scale`? We can use it if any.

Reply:
As far as I know, no scale method was found.